docs/extensibility.md

```diff
@@ -153,3 +153,86 @@ _Note:_ This filter must always be run on every page load, and not in your brows
 Extending the editor UI can be accomplished with the `registerPlugin` API, allowing you to define all your plugin's UI elements in one place.
 
 Refer to [the plugins module documentation](../plugins/) for more information.
+
+## Overriding autocompleters
```

Comment:
OK, I will move them in an `autocompletion.md` doc. Does that work for you?

Reply:
It all depends if you decide to have it as block specific or component specific :)

Here is the document for [Extending Blocks](https://github.com/WordPress/gutenberg/blob/master/docs/extensibility/extending-blocks.md).

It might be also its own `Extending Components` document. Autocompletion might be too granular.